Components/Button/button-component.css

```diff
@@ -49,3 +91,26 @@
 .btn-round {
 	border-radius: 1rem;
 }
+
+.btn i {
+	margin-right: 0.4rem;
+}
+
+.float-icon {
+	border-radius: 50%;
+	margin: 0;
+	height: 5.6rem;
+	width: 5.6rem;
+}
+
+.float-icon > i {
+	position: absolute;
+	top: 50%;
+	left: 50%;
+	transform: translateX(-50%) translateY(-50%);
+}
```

Comment:
An alternative which can be used;

```CSS
display: flex;
justify-content: center;
align-items: center;
```

or using Grid;
```CSS
display: grid;
place-items: center;
```

Reply:
Flex is much simpler, don't know how I forgot to use it